docker-compose.yaml

```diff
@@ -255,6 +255,7 @@ services:
     # EXTRA
     mysql:
         image: mysql:5.7
+        platform: linux/amd64
```

Comment:
In the future we should consider to remove this line and instead use a database that supports natively m1.

Reply:
ATM if we change to mysql 8.0 , we need to change the infrastructure images as well. We can approach that task globally in another moment :)

Reply:
I would say to change to another database if we do the change, postgres might be better since its open source and more powerful  than mysql.

Also notice that for database on pre/pro we do not use docker images AFAIK